pytracking/evaluation/tracker.py

```diff
@@ -256,6 +256,57 @@ def _store_outputs(tracker_out: dict, defaults=None):
         output['object_presence_score_threshold'] = tracker.params.get('object_presence_score_threshold', 0.55)
 
         return output
+        
+
+    def init_tracker(self, frame, optional_box, debug=None, visdom_info=None, save_results=False):
+        params = self.get_parameters()
+
+        debug_ = debug
+        if debug is None:
+            debug_ = getattr(params, 'debug', 0)
+        params.debug = debug_
+
+        params.tracker_name = self.name
+        params.param_name = self.parameter_name
+        self._init_visdom(visdom_info, debug_)
+
+        multiobj_mode = getattr(params, 'multiobj_mode', getattr(self.tracker_class, 'multiobj_mode', 'default'))
+
+        if multiobj_mode == 'default':
+            self.tracker = self.create_tracker(params)
+            if hasattr(tracker, 'initialize_features'):
+                self.tracker.initialize_features()
+
+        elif multiobj_mode == 'parallel':
```

Comment:
do we support this mode? Let's talk about our tracker after talking about pytracking in general

Reply:
ok

Reply:
the mode we run is parallel